### DISCUSSION TITLE
I take a photo of my school id and enrollment letter but still got rejected

### DISCUSSION THREAD

**Discussion by Georgefwt** [community/community#19664] [GitHub Education] [answered]
I took a photo of me holding my campus card and acceptance letter, while including the top half of my face, but I still got rejected. The email I recived said "Modifications to your photo proof might have been detected by the system". I don't understand why and there is no way I can modify that... The full version of reply like this:
```
The item you uploaded is insufficient to demonstrate your current academic status. Modifications to your photo proof might have been detected by the system. Please see suggestions.
Suggested proof to expedite your approval: 

•	School issued photo ID with current enrollment date. 

•	Schedule for current enrollment with school name.

•	Signed registration certificate with current enrollment date.

•	Transcripts with current enrollment date.
```
Could anyone give me suggestions, please? 🤝
By the way, I have tried 8 times. Will this makes me more difficult to be approved? :(

**Comment by melvincwng** [comment, marked as answer]
@Georgefwt Hi George, understand your frustrations in this issue.

For starters, try reapplying again using other student verification documents besides your student id. This can be your academic transcript, academic certifications, letter of enrolment to the school, school timetable, or even alternative student id cards (if you have others).

If all fails, my suggestion would be for you to raise a ticket to [GitHub education support](https://support.github.com/contact/education). Let the support engineers investigate your issue, and revert back to you.

Hope this reply addresses your issue.

If this reply has been useful, kindly appreciate if you could 'Mark as answer' so others know this issue has been addressed.

Take care, and all the best in your reapplication for the GitHub Student Developer Education pack 😊

**Reply by Georgefwt** [reply]
Thank you for your kind reply, I will try to submit some other proofs.😊
